org.adempiere.base/src/org/compiere/model/PO.java

```diff
@@ -1569,6 +1628,17 @@ else if (clazz == String.class)
 		return success;
 	}	//	load
 
+	/**
+	 * Load record with UUID
+	 * @param uuID - UUID
+	 * @param trxName - Transaction name
+	 */
+	protected void load(String uuID, String trxName)
+	{
+		load(0, trxName);
+		loadByUU(uuID, trxName);
```

Comment:
would prefer the following instead, change the original load(trxName) method to load(String uuid, String trxName) and add load(String trxName) {load(null, trxName);}. Do the empty uuid check above in the rename load(uuid, trxname) method.

Reply:
After refactoring, this method no longer required.